docs/shared/packages/powerpack-conformance/powerpack-conformance-plugin.md

```diff
@@ -183,17 +183,13 @@ To write your own conformance rule, specify a relative path to a TypeScript or J
 The rule definition file should look like this:
 
 ```ts {% fileName="tools/local-conformance-rule.ts" %}
-import type {
-  ConformanceRule,
-  ConformanceRuleResult,
-  createConformanceRule,
-} from '@nx/powerpack-conformance';
+import type { createConformanceRule } from '@nx/powerpack-conformance';
```

Comment:
This is a function, not a type, so import type should only be import

Reply:
Thanks, fixed in #28166 which I will cherry pick onto this branch before merge